spot-contracts/contracts/RolloverVault.sol

```diff
@@ -624,61 +716,64 @@ contract RolloverVault is
     //--------------------------------------------------------------------------
     // External & Public read methods
 
-    /// @inheritdoc IVault
-    function computeMintAmt(uint256 underlyingAmtIn) public view returns (uint256) {
-        //-----------------------------------------------------------------------------
-        uint256 feePerc = feePolicy.computeVaultMintFeePerc();
-        //-----------------------------------------------------------------------------
+    /// @notice Computes the amount of notes minted when given amount of underlying asset tokens
+    ///         are deposited into the system.
+    /// @param underlyingAmtIn The amount underlying tokens to be deposited into the vault.
+    /// @param feePerc The percentage of minted notes paid as fees.
+    /// @return noteAmtMint The amount of notes to be minted.
+    function computeMintAmt(uint256 underlyingAmtIn, uint256 feePerc) public view returns (uint256 noteAmtMint) {
```

Comment:
What's the motivation for replacing the line
`uint256 feePerc = feePolicy.computeVaultMintFeePerc();`
with the input parameter `feePerc`?